[2025-09-11T06:50:13.090+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-11T06:50:13.268+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: simple_daily_monitoring.check_data manual__2025-09-11T06:49:59.735427+00:00 [queued]>
[2025-09-11T06:50:13.308+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: simple_daily_monitoring.check_data manual__2025-09-11T06:49:59.735427+00:00 [queued]>
[2025-09-11T06:50:13.310+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-09-11T06:50:13.379+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): check_data> on 2025-09-11 06:49:59.735427+00:00
[2025-09-11T06:50:13.455+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'simple_daily_monitoring', 'check_data', 'manual__2025-09-11T06:49:59.735427+00:00', '--job-id', '5', '--raw', '--subdir', 'DAGS_FOLDER/monitor_dag.py', '--cfg-path', '/tmp/tmpe1ozxl60']
[2025-09-11T06:50:13.483+0000] {standard_task_runner.py:91} INFO - Job 5: Subtask check_data
[2025-09-11T06:50:13.578+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62 DeprecationWarning: This process (pid=320) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-09-11T06:50:13.592+0000] {standard_task_runner.py:64} INFO - Started process 336 to run task
[2025-09-11T06:50:14.888+0000] {task_command.py:426} INFO - Running <TaskInstance: simple_daily_monitoring.check_data manual__2025-09-11T06:49:59.735427+00:00 [running]> on host 02ae08abed04
[2025-09-11T06:50:16.017+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ops-team' AIRFLOW_CTX_DAG_ID='simple_daily_monitoring' AIRFLOW_CTX_TASK_ID='check_data' AIRFLOW_CTX_EXECUTION_DATE='2025-09-11T06:49:59.735427+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-11T06:49:59.735427+00:00'
[2025-09-11T06:50:16.024+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-11T06:50:16.092+0000] {logging_mixin.py:188} INFO - üìä Checking data health...
[2025-09-11T06:51:14.382+0000] {logging_mixin.py:188} INFO - ‚ö†Ô∏è Data is 2687.1 hours old
[2025-09-11T06:51:14.443+0000] {logging_mixin.py:188} INFO - Total records: 3137151
[2025-09-11T06:51:14.906+0000] {python.py:237} INFO - Done. Returned value was: stale_data
[2025-09-11T06:51:14.908+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-11T06:51:15.068+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=simple_daily_monitoring, task_id=check_data, run_id=manual__2025-09-11T06:49:59.735427+00:00, execution_date=20250911T064959, start_date=20250911T065013, end_date=20250911T065115
[2025-09-11T06:51:16.477+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-09-11T06:51:16.601+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-09-11T06:51:16.606+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
